ledger-tool/src/main.rs

```diff
@@ -1388,6 +1484,36 @@ fn main() {
                         .help("If snapshot creation should succeed with a capitalization delta."),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("simulate-block-production")
+                .about("Simulate producing blocks with banking trace event files in the ledger")
+                .arg(&load_genesis_config_arg)
+                .args(&accounts_db_config_args)
+                .args(&snapshot_config_args)
+                .arg(
+                    Arg::with_name("block_production_method")
+                        .long("block-production-method")
+                        .value_name("METHOD")
+                        .takes_value(true)
+                        .possible_values(BlockProductionMethod::cli_names())
+                        .help(BlockProductionMethod::cli_message()),
+                )
+                .arg(
+                    Arg::with_name("first_simulated_slot")
+                        .long("first-simulated-slot")
+                        .value_name("SLOT")
+                        .validator(is_slot)
+                        .takes_value(true)
+                        .required(true)
```

Comment:
Given trace files can we not figure out some reasonable default, i.e. the first traced leader slot?
and then leave this as an **optional** cli arg so they can delay the beginning of sim if desired.

Reply:
hehe, good question. for this, i need to figure out whether there's proper snapshot to replay...

Reply:
> Assume we're leaving this as-is for first iteration ?

I'm thinking yes.

> Seems like a nice to have, and can seemingly add something later if we come up with it

yes too.